tests/src/test/java/tachyon/client/FileSystemUtilsIntegrationTest.java

```diff
@@ -144,17 +145,19 @@ public void run() {
       @Override
       public void run() {
         try {
-          final TachyonConf conf = ClientContext.getConf();
           // set the slow default polling period to a more sensible value, in order
           // to speed up the tests artificial waiting times
-          conf.set(Constants.USER_FILE_WAITCOMPLETED_POLL_MS, "100");
-          // The write will take at most 600ms I am waiting for at most 400ms - epsilon.
-          boolean completed = FileSystemUtils.waitCompleted(sFileSystem, uri, 300,
-              TimeUnit.MILLISECONDS);
-          Assert.assertFalse(completed);
-          completed = sFileSystem.getStatus(uri).isCompleted();
-          Assert.assertFalse(completed);
-          ClientContext.reset();
+          ClientContext.getConf().set(Constants.USER_FILE_WAITCOMPLETED_POLL_MS, "100");
```

Comment:
Do you need to reinitialize afterwards? Usually when should `reinitialize` be called?